server/php-master.cpp

```diff
@@ -1493,20 +1467,23 @@ class QPSCalculator {
   }
 
   void update(double time_point, const PhpWorkerStats &new_worker_stats) {
-    if (!prev_worker_stats_.total_queries()) {
-      prev_worker_stats_ = new_worker_stats;
+    if (!prev_total_queries_) {
```

Comment:
I can't find, when this field is set to 0. I mean, while execution, when multiple requests are processed one by one

Reply:
It is set to 0 only on this struct initialization (1502 line), and then it is only incrementally updated.

Reply:
It's not annulated between requests. Is it on purpose?

Reply:
Yes, it is ok. new_worker_stats.total_queries() returns the monotonically incremented requests count, so we get a delta between the old value and the new one.